### DISCUSSION TITLE
Como estudar para python

### DISCUSSION THREAD

**Discussion by RenataMachado** [frontendbr/forum#2478] [Perguntas e Respostas]
Oi gente, vocês tem alguma indicação de material para python ? Além de vídeos do YouTube.

**Comment by nazarepiedady** [comment]
@RenataMachado, lamento que não tenhas tido uma resposta a tempo, mas apesar de teres pedido algo além da YouTube, gostaria que reconsiderasses o material do nosso colega @henriquebastos com o seu canal [HB Network](https://www.youtube.com/@hbnetworkoficial).

Se ainda assim, preferires algo fora da YouTube, podes considerar o material do [dev.pro.br](https://www.dev.pro.br/)

**Comment by rafgam10** [comment]
opa man, tudo bem? pegar um playlist de python normal mesmo, com os seguintes assuntos variaveis, if e else (estruturas condicionais), for e while (estruturas de repetição), listas, tuplas, sets, dicionarios, funções, POO (Programação Orientada a Objetos, link="https://www.youtube.com/@ProgramadorLhama/playlists") e após isso vc decide oq vc quer estudar, se vai para análise de dados, utilização de interface-gráficas para criar software, ou ir para programação web utilizando django, criação de API com flask e etc...

**Reply by rafgam10** [reply]
espero que eu tenha ajudando, mas este link de canal tem quase tudo, mas qualquer coisa vc pode seguir o guia que eu digitei para ficar melhor, para cada passo, faça exercícios pode utilizar chatGPT para ele criar exercício sobre cada tema, boa sorte man sucesso!
